module/settings.mjs

```diff
@@ -339,6 +339,22 @@ export function registerSystemSettings() {
     type: Boolean,
     default: true
   });
+
+  if (game.release.generation >= 12) {
+    game.settings.register("dnd5e", "defaultSkills", {
+      name: "SETTINGS.5eDefaultSkillsN",
+      hint: "SETTINGS.5eDefaultSkillsH",
+      type: new foundry.data.fields.SetField(
+        new foundry.data.fields.StringField({
+          choices: () => CONFIG.DND5E.skills
+        }), {
+          type: "checkboxes"
```

Comment:
(This doesn't do anything. I left it here just to name and shame.)

Reply:
Is there a core issue filed for this? Probably just a result of the Settings app still being Application V1.

That said, I'm not sure checkboxes would be right here since there are so many skills, it might result in a big wall of checkboxes.

Reply:
There is not. Though I agree in hindsight it is also best to leave it out until we know what the SettingsConfigurationV2 will look like.